src/core/Akka/Configuration/Hocon/HoconValue.cs

```diff
@@ -73,6 +73,11 @@ public bool IsEmpty
         /// </summary>
         internal bool AdoptedFromFallback { get; private set; }
 
+        public Config ToConfig()
+        {
+            return new Config(new HoconRoot(this, Enumerable.Empty<HoconSubstitution>()));
```

Comment:
Is the idea here to perform a deep, immutable clone of the entire HOCON graph?

Reply:
No, its just a convenience method to convert a `HoconValue` into a `Config`